<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="primaryViewerWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="view"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpenImageFile"/>
    <addaction name="actionOpenPathPlanningConfig"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionAdjustView"/>
    <addaction name="actionSaveCurrentView"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImageFile"/>
   <addaction name="actionOpenPathPlanningConfig"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionAdjustView"/>
   <addaction name="actionSaveCurrentView"/>
  </widget>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../control-map-viewer/icons.qrc">
     <normaloff>:/resources/icons/ic_zoom_in_black_24dp.png</normaloff>:/resources/icons/ic_zoom_in_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in the image</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../control-map-viewer/icons.qrc">
     <normaloff>:/resources/icons/ic_zoom_out_black_24dp.png</normaloff>:/resources/icons/ic_zoom_out_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out the image</string>
   </property>
  </action>
  <action name="actionAdjustView">
   <property name="icon">
    <iconset resource="../control-map-viewer/icons.qrc">
     <normaloff>:/resources/icons/ic_settings_overscan_black_24dp.png</normaloff>:/resources/icons/ic_settings_overscan_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>AdjustView</string>
   </property>
   <property name="toolTip">
    <string>Adjusts the view to the available area</string>
   </property>
  </action>
  <action name="actionOpenImageFile">
   <property name="icon">
    <iconset resource="../control-map-viewer/icons.qrc">
     <normaloff>:/resources/icons/ic_folder_open_black_24dp.png</normaloff>:/resources/icons/ic_folder_open_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Open image file</string>
   </property>
  </action>
  <action name="actionOpenPathPlanningConfig">
   <property name="icon">
    <iconset resource="../control-map-viewer/icons.qrc">
     <normaloff>:/resources/icons/ic_keyboard_black_24dp.png</normaloff>:/resources/icons/ic_keyboard_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Open path planning config</string>
   </property>
   <property name="toolTip">
    <string>Open path planning config</string>
   </property>
  </action>
  <action name="actionSaveCurrentView">
   <property name="icon">
    <iconset resource="../control-map-viewer/icons.qrc">
     <normaloff>:/resources/icons/ic_camera_enhance_black_24dp.png</normaloff>:/resources/icons/ic_camera_enhance_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Save current view</string>
   </property>
   <property name="toolTip">
    <string>Save current view</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../control-map-viewer/icons.qrc"/>
 </resources>
 <connections/>
</ui>
